shared/src/simulation/activities/reassign-treatments.ts

```diff
@@ -380,7 +377,7 @@ function createCateringPersonnel(
 ): CateringPersonnel[] {
     return personnel.map((pers) => ({
         personnel: pers,
-        priority: personnelPriorities[pers.personnelType],
+        priority: personnelPriorities[pers.personnelType] ?? 0,
```

Comment:
Use `-1`, since `0` is an actually used priority:


```suggestion
        priority: personnelPriorities[pers.personnelType] ?? -1,
```